pkg/operation/botanist/secrets.go

```diff
@@ -903,6 +965,32 @@ func (b *Botanist) SyncShootCredentialsToGarden() error {
 	return nil
 }
 
+// SyncKibanaUserCredentialsToGarden copies the kibana user "user" to
+// the project namespace in the Garden cluster.
+func (b *Botanist) SyncKibanaUserCredentialsToGarden() error {
```

Comment:
See above: Remove this function - take a look how it's done here: https://github.com/gardener/gardener/pull/1000/files#diff-a68e0b6e59734836dbbde01299666e87R895